### ISSUE TITLE
Something wrong when send to Repeater

### ISSUE THREAD

**Issue by burper00** [doyensec/inql#135]
Hello, I have a problem with send InQL to send Repeater, sending post after response is blank screen I show you whats the problem.

![1](https://github.com/doyensec/inql/assets/157926181/eca4ee4d-d636-4ae2-a695-aa6b0b32262d)
![2](https://github.com/doyensec/inql/assets/157926181/bac8df55-4aec-4b58-bc7c-3253047173a3)
![3](https://github.com/doyensec/inql/assets/157926181/d1628aaa-2878-4ecd-a410-486eb26fdbeb)
![4](https://github.com/doyensec/inql/assets/157926181/578b2d66-ce25-4a2f-829d-bc6f4e344ac7)

Everything is fine when it comes to send Repeater... Click Send after response is black screen. Its says me like error or something like this on pic about like ' change it to HTTP/1.1' but when I make it its giving to me protocol error...

 Actually is about when InQL to send repeater it should be HTTP/2 protocol.  But this InQL sends in HTTP/1.1 protocol.

I updated to my burp last version and reinstall inQL but same problem

**Desktop (please complete the following information):**
 - OS: Windows10 64 Bit
 - Java Version: java version "17.0.9" 2023-10-17 LTS
Java(TM) SE Runtime Environment (build 17.0.9+11-LTS-201)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.9+11-LTS-201, mixed mode, sharing)
- Python Version: Python 3.12.1
- Burp Version: 12.1.3
 - Version: 23

**Comment by execveat** [comment]
Hey @burper00, thank you for reporting the bug! It seems to be fixed in our development branch, and we are going to release the next version shortly. I'll ping you when the 5.1 is available for testing.

<img width="856" alt="image" src="https://github.com/doyensec/inql/assets/105389353/f36b9bc2-21f0-44a9-add1-14271ff691da">

**Comment by burper00** [comment]
Thanks 👍

**Comment by bartek-doyensec** [comment]
We are preparing to release the v6.0 update, which rewrites InQL into Kotlin language. This major update should address the issue, so we are closing it to help clear the backlog.

If you believe this issue still exists in the latest version (v6.0), please feel free to create a new issue.
Thanks!

**Comment by burper00** [comment]
![Image](https://github.com/user-attachments/assets/4c084996-b42a-4dd3-a640-972e07e6a5b7)

That's awesome, thank you so much! The new version is amazing! And from what I can see, the speed and stability have also improved. Great work! 👏

Her şey harika çalışıyor, fakat eklentiyi yüklerken bunun gibi uyarılar çıkıyor. Normal mi acaba ?

Sistemim windows 10 64 bit ve güncel. 

![Image](https://github.com/user-attachments/assets/355efaba-5b55-41a0-ac85-4fe328899916)

Java version
java 22.0.1 2024-04-16
Java(TM) SE Runtime Environment (build 22.0.1+8-16)
Java HotSpot(TM) 64-Bit Server VM (build 22.0.1+8-16, mixed mode, sharing)

**Comment by bartek-doyensec** [comment]
Yes, this is perfectly normal. We log some things for easier debugging. 
In your case, the logs says that the save state was not found because it is a fresh install. When you work on some project and use InQL it will save queries and batch attacker history and it will use it on next load.

**Comment by burper00** [comment]
Thank you so much! I'm really grateful for this, I find the new version amazing.
